spec/models/bulkrax/csv_entry_spec.rb

```diff
@@ -38,14 +38,13 @@ module Bulkrax
         before do
           allow_any_instance_of(ObjectFactory).to receive(:run)
 
-          allow(subject).to receive(:record).and_return('source_identifier' => '3', 'title' => 'some title', 'file' => './spec/fixtures/csv/test file.csv')
+          allow(subject).to receive(:record).and_return('source_identifier' => '3', 'title' => 'some title')
           allow(File).to receive(:exist?).with('./spec/fixtures/csv/test_file.csv').and_return(true)
         end
 
         it 'sets up the file_path and removes spaces from filenames' do
           subject.build
           expect(subject.status).to eq('succeeded')
-          expect(subject.parsed_metadata['file']).to eq(['./spec/fixtures/csv/test_file.csv'])
```

Comment:
@orangewolf I believe the point of this spec was to make sure that if a file was uploaded that had spaces in the filename, that those spaces would get replaced with underscores. Is that no longer needed? If so, should the whole spec be removed?

Reply:
Hmm, I wonder why it was broken then. I’ll look at it

Reply:
you were right - this was a bug... now it is not